packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -434,6 +436,9 @@ export class GarbageCollector implements IGarbageCollector {
 	// This is the version of GC data in the latest summary being tracked.
 	private latestSummaryGCVersion: GCVersion;
 
+	// Feature Support info persisted to this container's summary
+	private readonly persistedGcFeatureSupportInfo: GCFeatureMatrix | undefined;
```

Comment:
```suggestion
	private readonly persistedGcFeatureMatrix: GCFeatureMatrix | undefined;
```

Reply:
Will do in subsequent PR that flips disable -> allow nomenclature